source/_integrations/triggercmd.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: TRIGGERcmd
+description: How to integrate TRIGGERcmd with Home Assistant.
+ha_category:
+  - Switch
+  - Automation
+ha_release: "2024.10"
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@rvmey'
+ha_domain: triggercmd
+ha_platforms:
+  - switch
+ha_zeroconf: false
+ha_integration_type: integration
+---
+
+The TRIGGERcmd {% term integration %} allows you to run commands on computers via [TRIGGERcmd](https://triggercmd.com/) with Home Assistant.
+
+
+## Prerequisites
+
+To use TRIGGERcmd, you need the following:
+
+- A [TRIGGERcmd account](https://www.triggercmd.com/user/auth/signup)
+- A computer with the TRIGGERcmd agent running on it
+- For instructions on installing and using TRIGGERcmd itself, refer to the [TRIGGERcmd Quick Start Guide](https://docs.triggercmd.com/#/./QuickStart).
```

Comment:
```suggestion
- For instructions on installing and using TRIGGERcmd itself, refer to the [TRIGGERcmd Quick Start Guide](https://docs.triggercmd.com/#/./QuickStart).
- after adding the TRIGGERcmd integration, you will be prompted for your **user token**. This can be found at the bottom of your TRIGGERcmd profile page or the instructions page.
```